### PR TITLE
feat(uwg): Add CI docs.

### PR DESCRIPTION
**Pull request by saeranv**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,40 +1,28 @@
...
+    author="Ladybug Tools",
+    author_email="info@ladybug.tools",
+    description="The Urban Weather Generator (uwg) is a Python application for modeling the urban heat island effect.",
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/ladybug-tools/uwg",
-    packages=['uwg'],
-    include_package_data=True,
+    packages=setuptools.find_packages(exclude=["tests*"]),
+    install_requires=requirements,
```
Comment (chriswmackey) [inline setup.py:20]: You need to set `include_package_data=True` like you see here:
https://github.com/ladybug-tools/honeybee-standards/blob/master/setup.py#L18

File: setup.py [hunk 2]
```diff
@@ -1,40 +1,28 @@
...
+    setup_requires=['setuptools_scm'],
+    author="Ladybug Tools",
+    author_email="info@ladybug.tools",
+    description="The Urban Weather Generator (uwg) is a Python application for modeling the urban heat island effect.",
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/ladybug-tools/uwg",
-    packages=['uwg'],
-    include_package_data=True,
+    packages=setuptools.find_packages(exclude=["tests*"]),
```
Comment (chriswmackey) [inline setup.py:19]: You probably also want to exclude your resources folder.

File: tests/example_test.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import pytest
```
Comment (chriswmackey) [inline tests/example_test.py:1]: You can delete this whole file.  It's just a placeholder form the cookiecutter template

File: tests/conftest.py [hunk 4]
```diff
@@ -0,0 +1 @@
+import pytest
```
Comment (chriswmackey) [inline tests/conftest.py:1]: You can also remove this file.

File: setup.py (hunk 2 above)
Comment (saeranv) [reply setup.py:19]: Like this: 
`packages=setuptools.find_packages(exclude=["tests*", "resources"]),`

?

**Pull request merged**